source/options.tsx

```diff
@@ -129,6 +129,11 @@ async function clearCacheHandler(event: Event): Promise<void> {
 	}, 2000);
 }
 
+async function bisectFeaturesHandler(): Promise<void> {
+	await cache.set('bisect', [], {minutes: 5});
+	await browser.tabs.create({url: 'https://github.com'});
```

Comment:
Ending up on the homepage is a little confusing because it immediately asks the user if they can see the feature. Instead, in this page we should just instruct the user to visit the page where they're seeing the feature and refresh it to see the new instructions on screen.

Ideally this is done via `alert()` but it might not work on Firefox. In that's the case, let's just add a description of this in the HTML file (a new `<p>`)